extensions/functions_arithmetic.yaml

```
@@ -284,32 +284,84 @@ scalar_functions:
             value: fp64
         return: fp64
   -
-    name: "modulus"
-    description: "Get the remainder when dividing one value by another."
+    name: "modulo"
+    description: >
+      Calculate the remainder (r) when dividing dividend (x) by divisor (y).
+
+      In mathematics, many conventions for the modulo operation exists. In computing the result of a modulo operation
```

```suggestion
      In mathematics, many conventions for the modulo operation exists. The result of a modulo operation
```
This read a little odd to me.  I wasn't sure if that is "in the field of computing, the result..." or "while computing the result...".  I think simpler might be better here.